tox.ini

```diff
@@ -4,8 +4,8 @@ max_line_length=118
 [tox]
 envlist =
     {py38,py39,py310,py311}-django{42}-crispy{2,-latest},
-    {py310,py311,py312}-django{50,51,-latest}-crispy{2,-latest},
```

Comment:
It seems that tests never actually testing against 'latest'. Rather the latest release on PyPI. 🤦 

Anyway, tests fail against 5.2 / django main branch at the moment so those tests need to be skipped. 

For now, I think it's easiest to remove this and can always add it back in future.